docs/AutomationReference.md

```diff
@@ -123,26 +123,57 @@ The `ScannerFactory.CreateScanner` method returns an `IScanner` object.
 
 #### IScanner
 
-##### `Scan`
-The Scan runs AxeWindows automated tests using the config provided at the time of creation of the scanner.
+##### `ScanAsync`
+The `ScanAsync` method asynchronously runs AxeWindows automated tests using the config provided at the time of creation of the scanner. This method should be `await`ed.
 
 ###### Parameters
+`IScanner.ScanAsync` accepts two parameters:
+
+**Name** | **Type** | **Description**
+---|---|---
+`scanOptions` | `ScanOptions` | An object describing custom settings for this scan. Pass `null` for default options.
+`cancellationToken` | `CancellationToken` | A [`CancellationToken`](https://learn.microsoft.com/en-gb/dotnet/api/system.threading.cancellationtoken?view=netstandard-2.0).
 
-The `Scan` method accepts 0 or 1 parameters.
+###### Return object
+`ScanAsync` returns a `ScanOutput` object.
+
+##### `Scan`
+The `Scan` method synchronously runs AxeWindows automated tests using the config provided at the time of creation of the scanner, and blocks until the scan is complete.
+
+###### Parameters
+`IScanner.Scan` accepts one parameter:
+###### Parameters
+`IScanner.ScanAsync` accepts two parameters:
 
 **Name** | **Type** | **Description**
 ---|---|---
-scanId | `string` | A string identifier for the scan. If the scan produces output files based on the `Config` object used to create the scanner, the output files will be given the name of the scan id (e.g., MyScanId.a11ytest).
+`scanOptions` | `ScanOptions` | An object describing custom settings for this scan. Pass `null` for default options.
 
 ###### Return object
+`Scan` returns a `ScanOutput` object.
 
-The `Scan` method returns a `ScanResults` object and has the following properties:
+#### `ScanOptions`
+The `ScanOptions` constructor accepts the following arguments:
+
+**Name** | **Type** | **Description** | **Default value**
+---|---|---
```

Comment:
This table is malformed because we have more column headers than columns defined. I think this needs to be
```suggestion
---|---|---|---
```